opensrp-anc/src/main/java/org/smartregister/anc/library/constants/AncFormConstants.java

```diff
@@ -0,0 +1,18 @@
+package org.smartregister.anc.library.constants;
+
+public class AncFormConstants {
+
+    public static final class SpinnerKeyConstants {
+        public static final String PROVINCE = "province";
+        public static final String DISTRICT = "district";
+        public static final String SUB_DISTRICT = "subdistrict";
+        public static final String FACILITY = "health_facility";
+        public static final String VILLAGE = "village";
+    }
+
+    public static final class FormKeyConstants {
+        public static final String SUB_TYPE = "sub_type";
+        public static final String LOCATION_SUB_TYPE = "location";
+    }
+
+}
```

Comment:
I think we already have other constant files in the app. Please move these classes in there.

Reply:
This is resolved